tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_homescreen_change_wallpaper.py

```diff
@@ -18,7 +20,8 @@ def test_homescreen_change_wallpaper_from_gallery(self):
         https://moztrap.mozilla.org/manage/case/1902/
         reusing /functional/homescreen/test_homescreen_change_wallpaper.py script
         """
-
+        # wait until the homescreen is fully drawn
+        time.sleep(3)
```

Comment:
I was looking into it a bit, but I think the icon were being detected in DOM while not fully drawn, I'll look for any other markers that would indicathe this.